src/safariservices.cs

```diff
@@ -171,6 +176,15 @@ interface SFSafariExtensionHandling
 
 		[Export ("popoverViewController")]
 		SFSafariExtensionViewController PopoverViewController { get; }
+
+		[Mac (10, 12, 4)]
+		[Async (ResultTypeName="SFValidationResult")]
+		[Export ("validateContextMenuItemWithCommand:inPage:userInfo:validationHandler:")]
+		void ValidateContextMenuItem (string command, SFSafariPage page, [NullAllowed] NSDictionary<NSString, NSObject> userInfo, Action<bool, NSString> validationHandler);
```

Comment:
Please create a delegate type instead of `Action<bool, NSString>` (and drop the `ResultTypeName`). That should be the case for > 1 parameter, unless the 2nd one is an `NSError`.

The former is easier to use, because the parameters are named. The later requires extra documentation to be useful.

There will soon be a test to catch those (at least in master) once I'm finished adding missing `Async`.